lib/error-matcher.js

```diff
@@ -52,13 +57,24 @@ module.exports = (function () {
         initialColumn: col,
         searchAllPanes: true
       });
+      this.emit('scroll', match.type, match.id);
     }.bind(this));
   };
 
   ErrorMatcher.prototype._parse = function () {
-    this.currentMatch = XRegExp.forEach(this.output, this.regex, function (match) {
+    this.currentMatch = XRegExp.forEach(this.output, this.regex, function (match, i) {
+      match.type = 'error';
+      match.id = 'error-' + i;
       this.push(match);
     }, []);
+    var output = _.escape(this.output);
+    var matchIndex;
+    for (matchIndex in this.currentMatch) {
+      var search = _.escape(this.currentMatch[matchIndex][0]);
+      var replace = '<a class=\"' + this.currentMatch[matchIndex].type + '\" id=\"' + this.currentMatch[matchIndex].id + '\">$&</a>';
```

Comment:
Something odd probably happens in the replace here, as it seems as the newline character right after the `Executing: ...` line is removed here.